Code/GraphMol/Descriptors/ConnectivityDescriptors.h

```diff
@@ -172,6 +172,14 @@ const std::string kappa2Version = "1.1.0";
 RDKIT_DESCRIPTORS_EXPORT double calcKappa3(const ROMol &mol);
 const std::string kappa3Version = "1.1.0";
 
+//! calculate the Kier Phi value for a molecule
+//! From Quantitative Structure-Activity Relationships 8, 221–224 (1989).
```

Comment:
Perhaps the hybridization causing diffs between this and the paper should be noted here.

Reply:
It's not just here. This particular mismatch occurs for Hall-Kier Alpha and all the descriptors which are derived from it: kappa1, kappa2, kappa3, and now phi.
I'll update the other comments as well